validate/validate.go

```diff
@@ -128,6 +140,11 @@ func (v *Validator) CheckRootfsPath() (msgs []string) {
 		msgs = append(msgs, fmt.Sprintf("The root path %q is not a directory.", rootfsPath))
 	}
 
+	rootParent := filepath.Dir(absRootPath);
+	if rootParent == string(filepath.Separator) || rootParent != absBundlePath {
```

Comment:
> You mean rootParent can't be "/"?

That should be fine.  The `Separator` comparison is intended to catch  `absRootPath` being a spelling of `/`, based on [the `Dir` docs][1]:

> The returned path does not end in a separator unless it is the root directory.

But I'll double check `Dir("/") ` later tonight.

[1]: https://golang.org/pkg/path/filepath/#Dir